src/jit/machine.rs

```diff
@@ -284,7 +284,7 @@ impl BaselineJitMachine {
                 } else {
                     // TODO: check if dynasm handles alignments here.
                     emitter.encode(&mut buffer_mut[rust_data.used..])?;
-                    let offset = rust_data.used;
+                    let offset = rust_data.used as u64;
```

Comment:
I notice the `used` field is still usize type, not sure if this is missed or is intentioned.

Reply:
Yes this is kept as usize for 2 reasons:

1. Actually [here](https://github.com/nervosnetwork/ckb-vm/blob/2ffd6f12b750949c88103f26eb2b27e762ff1d8f/src/jit/machine.rs#L281) we ensure that used can never exceed 1MB, which will fit well within usize even for 32-bit platform
2. The whole JIT package right now is still treated as a PoC and won't be included in production, in fact, I'm thinking of removing the whole JIT package once the new AOT mode is in place, that's why this PR only contains necessary fixes in the JIT package, and still many fields in JIT package as usize.